socrata/socrata_upload.py

```diff
@@ -172,31 +176,56 @@ def upload(method, asset_id, sql_query, overwrite, year=None, township=None):
     input_data[date_columns] = input_data[date_columns].map(
         lambda x: x.strftime("%Y-%m-%dT%X")
     )
-    input_data = input_data.to_json(orient="records")
 
     # Raise URL status if it's bad
-    requests.get(
-        url=url,
-        data=input_data,
-        auth=auth,
+    s.get(
+        (
+            "https://datacatalog.cookcountyil.gov/resource/"
+            + asset_id
+            + ".json?$limit=1"
+        ),
+        headers={"X-App-Token": os.getenv("SOCRATA_APP_TOKEN")},
```

Comment:
**[Question, non-blocking]** Also, why do we use the `headers` kwarg here, while we template the token directly into the `url` string on line 147? I'm inclined to prefer `headers` in both cases but I don't totally understand the tradeoff.